Server/src/cli/CLI_USAGE_GUIDE.md

```
@@ -0,0 +1,727 @@
+cd /path/to/unity-mcp/Server
+uv run mcp-for-unity --transport http --http-url http://localhost:8080
+```
+
+### 2. Verify Connection
+
+```bash
+unity-mcp status
+```
+
+Expected output:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)